api/hypershift/v1beta1/nodepool_types.go

```diff
@@ -705,6 +706,25 @@ type KubevirtNodePoolPlatform struct {
 	// +optional
 	// +kubebuilder:validation:Enum=Enable;Disable
 	NetworkInterfaceMultiQueue *MultiQueueSetting `json:"networkInterfaceMultiqueue,omitempty"`
+
+	// NetworkNames specify the networks attached to the nodes, they can be special value "default" with means pod network
+	// or value with format "[namespace]/[name]" to reference at multus network attachment definition
+	//
+	// +optional
+	Networks []KubevirtNetwork `json:"networks"`
+}
+
+// KubevirtNetwork specifies the configuration for a virtual machine
+// network interface interface
+type KubevirtNetwork struct {
+	// Name specify the network attached to the nodes, it can be the special value "default" with means pod network
+	// or a value with format "[namespace]/[name]" to reference at multus network attachment definition
+	Name string `json:"name"`
+
+	// Model Interface model. One of: e1000, e1000e, ne2k_pci, pcnet, rtl8139, virtio. Defaults to virtio.
```

Comment:
this comment stutters.

Reply:
Model is not longer at this PR, I will resolve this.